next.json

```diff
@@ -105,6 +105,78 @@
             {"value": 3, "name": "storage buffer"}
         ]
     },
+    "blend factor": {
+        "category": "enum",
+        "values": [
+            {"value": 0, "name": "zero"},
+            {"value": 1, "name": "one"},
+            {"value": 2, "name": "src color"},
+            {"value": 3, "name": "one minus src color"},
+            {"value": 4, "name": "src alpha"},
+            {"value": 5, "name": "one minus src alpha"},
+            {"value": 6, "name": "dst color"},
+            {"value": 7, "name": "one minus dst color"},
+            {"value": 8, "name": "dst alpha"},
+            {"value": 9, "name": "one minus dst alpha"},
+            {"value": 10, "name": "src alpha saturated"},
+            {"value": 11, "name": "blend color"},
+            {"value": 12, "name": "one minus blend color"},
+            {"value": 13, "name": "src1 color"},
```

Comment:
Or keep it because you implemented tests and all for it, but be ready to remove it / make it an extension later.